### PR TITLE
Manja14 Spectral Ingest 

### PR DESCRIPTION
**Pull request by SherelynA**

### CODE REVIEW

**Comment by kelle** [comment]
I figure out why the spectra aren't being ingested!  It's because the telescope, instrument, and modes need to be added. @SherelynA , this is your next step. Add ISSAC, SW LRES, and ESO VLT U3.

[ISAAC was on VLT U3](https://www.eso.org/sci/facilities/paranal/decommissioned/isaac.html) so we should add "ESO VLT U3" as a telescope rather than just "VLT". (VLT is actually 4 different telescopes.)

File: data/2mass_j12451416-4429077.json [hunk 1]
```diff
@@ -243,6 +246,20 @@
             "wavelength_order": null,
             "comments": null,
             "reference": "Loop07b"
+        },
+        {
+            "spectrum": "\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000",
```
Comment (kelle) [inline data/2mass_j12451416-4429077.json:251]: Uh-oh. This is a problem. Not sure what's going on here. What's the spectrum supposed to be?

File: data/cha_130540-773958.json [hunk 2]
```diff
@@ -14,6 +14,25 @@
...
+            "local_spectrum": null,
+            "regime": "nir",
+            "telescope": "ESO VLT U3",
+            "instrument": "ISAAC",
+            "mode": "SW LRes",
+            "observation_date": "2010-04-21T00:00:00",
+            "wavelength_units": null,
+            "flux_units": null,
+            "wavelength_order": null,
+            "comments": "April 6, 2010",
```
Comment (kelle) [inline data/cha_130540-773958.json:34]: The comment with an additional observation date should have more words. For example, "spectrum also includes data observed on April 6, 2010."

**Review by kelle — CHANGES_REQUESTED** [review]

File: data/cha_130540-773958.json (hunk 2 above)
Comment (SherelynA) [reply data/cha_130540-773958.json:34]: I made the comment more comprehensive in the new csv.

**Review by kelle — APPROVED** [review]

**Pull request merged**